packages/@storefront/flux-capacitor/src/core/store/index.ts

```diff
@@ -55,7 +55,7 @@ namespace Store {
     area: string;
     query: Query; // mixed
 
-    sorts: SelectableList<Sort>;
+    sorts: LabelSelectableList<Sort>;
```

Comment:
Maybe `SelectableListWithLabels` or `LabeledSelectableList`. `Label-` doesn't really tell me much about what the object is.

Reply:
Good call, will update.